README.md

```diff
@@ -733,20 +732,29 @@ window.addEventListener("MSSDK:redeem-coupon-failed", evt =>
+| `MSSDK:unsubscribe-action-confirmed`    | `{cancellationReason: "Selected reason", offerId: "S123456789_US"}`                      | The event will be emitted after clicking confirm unsubscribe button in my account. This button cancels the subscription.                                                                                                                                                                                                                                                                                                              |
+| `MSSDK:unsubscribe-action-cancelled`    | `null`                                                                                   | The event will be emitted after clicking unsubscribe cancellation button in my account unsubscribe popup. This button cancels the unsubscribe process.                                                                                                                                                                                                                                                                                |
+| `MSSDK:resume-button-clicked`           | `{offerId: "S123456789_US"}`                                                             | The event will be emitted after clicking the resume button in my account. This button opens a resume pop up.                                                                                                                                                                                                                                                                                                                          |
+| `MSSDK:resume-action-confirmed`         | `{offerId: "S123456789_US"}`                                                             | The event will be emitted after clicking confirm resume button in my account. This button reactivates the subscription.                                                                                                                                                                                                                                                                                                               |
+| `MSSDK:resume-action-cancelled`         | `null`                                                                                   | The event will be emitted after clicking confirm resume cancellation button in my account resume subcription popup. This button cancels the subscription reactivation process.                                                                                                                                                                                                                                                        |
```

Comment:
I would simplify the description to `This event will be emitted when the user resigns to resume the subscription.` and optional: `This button cancels the subscription reactivation process and closes the resume popup`